cvat/apps/engine/media_extractors.py

```diff
@@ -283,8 +312,9 @@ def get_image_size(self, i):
         return image.width, image.height
 
 class IChunkWriter(ABC):
-    def __init__(self, quality):
+    def __init__(self, quality, dimension=DimensionType.DIM_2D):
```

Comment:
Again better to inherit here. For example, IChunkWriter for common code, IChunk2dWriter(IChunkWriter) and IChunk3dWriter(IChunkWriter) or something like that.

Reply:
The only parameter we require for distinguishing 2D and 3D task is _dimension which by default is always 2D.
Only in the case of zipCompressedChunkWriter we require the same and only at this point we have an `if`. 
If another class is created for the same, such as zipCompressed3DChunkWriter along with IChunk3dWriter, then we require to place multiple `if` in frameProvider and task.py. Please let me know if you have any different opinion.

Reply:
Does an additional type that is defined here https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/engine/models.py#L34-L37 solve this problem?

Reply:
Hi @azhavoro,

The additional type mentioned here will not help since the ext type 'zip' can have both images and pcd data and the additional type list accepts list as input whereas we have zip as input.